app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -17,70 +17,68 @@
 
   <!-- Editing grant -->
   <div v-else-if="grant" class="flex flex-col justify-center sm:px-6 lg:px-8">
-    <div class="sm:mx-auto sm:w-full sm:max-w-md">
-      <img
-        class="mx-auto h-12 w-auto"
-        src="https://tailwindui.com/img/logos/workflow-mark-indigo-600.svg"
-        alt="Workflow"
-      />
-      <h2 class="mt-6 text-center text-3xl font-extrabold text-gray-900">Edit Grant {{ grantMetadata?.name }}</h2>
+    <div class="sm:mx-auto sm:w-full mt-5 md:mt-20 mb-12 md:mb-24">
+      <h1>Edit Grant<br />"{{ grantMetadata?.name }}"</h1>
```

Comment:
Nit, but maybe this instead of `<br/>`? Personal preference, but more clear IMO

```suggestion
      <h1>Edit Grant</h1>
      <h1>"{{ grantMetadata?.name }}"</h1>
```